xroad-catalog-persistence/src/main/java/fi/vrk/xroad/catalog/persistence/entity/OrganizationDescription.java

```diff
@@ -45,7 +45,7 @@
     private String language;
     @Column(nullable = false)
     private String type;
-    @Column(nullable = false)
+    @Column(nullable = false, length = 10000)
```

Comment:
what is the length for?

Reply:
I mean should be enough for sure, but is necessary or should it be smaller that 10000?

Reply:
I am not sure about the exact maximum needed here, it was the default of 255, but this was not enough as some names and descriptions were 260 and more in length, I could set it also to 1000.

Reply:
WSDL has 4000 for example